### ISSUE TITLE
test.py script question

### ISSUE THREAD

**Issue by as7438** [cchen-cc/MA-SAM#17]
Hi, 

I was wondering in test.py why there's a specified HU_min and HU_max, as well as data_mean and data_std. Moreover, there is some added np.clip and normalization done to the image and masks. Assuming our test set is alraedy pre-processed the same as training set, is there any reason to use these additional steps?
